Helper/FileSystem.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * This file is part of the EzSystemRecommendationBundle package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Helper;
+
+use EzSystems\RecommendationBundle\Authentication\Authenticator;
+
+class FileSystem
+{
+    /** @var \EzSystems\RecommendationBundle\Authentication\Authenticator */
+    protected $authenticator;
+
+    /**
+     * FileSystem constructor.
+     * @param \EzSystems\RecommendationBundle\Authentication\Authenticator $authenticator
```

Comment:
I would remove that comment altogether. It doesn't have more value than the method's prototype.